packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [57.0.0]
```

Comment:
you know what, I really appreciate adding the changelogs updates in previous PRs 🎉

Reply:
Yes, it makes releasing a much simpler task. No need to go through several PRs and making up the changelog at the last minute.